core/string/ustring.cpp

```diff
@@ -3538,15 +3540,18 @@ String String::format(const Variant &values, const String &placeholder) const {
 				} else {
 					ERR_PRINT(String("STRING.format Inner Array size != 2 ").ascii().get_data());
 				}
-			} else { //Array structure ["RobotGuy","Logis","rookie"]
-				Variant v_val = values_arr[i];
-				String val = v_val;
+				continue;
```

Comment:
Should this be a 'break' or can we use both 'Array' and 'nested Array' in the same format call?

Reply:
You always could mix the two (_for some reason_).